thoth/adviser/state.py

```diff
@@ -44,6 +44,19 @@ class State:
     resolved_dependencies = attr.ib(default=attr.Factory(dict))  # type: Dict[str, Tuple[str, str, str]]
     _parent = attr.ib(default=None)  # type: weakref.ReferenceType['State']
     advised_runtime_environment = attr.ib(type=Optional[RuntimeEnvironment], kw_only=True, default=None)
+    # an example for a manifest change:
```

Comment:
is this inspired or just like how kustomize uses jsonpatch? see https://github.com/kubernetes-sigs/kustomize/blob/master/examples/jsonpatch.md

Reply:
it's inspered, however we could use https://pypi.org/project/jsonpatch/ might be a better option